getting_started/step_by_step/exporting.rst

```diff
@@ -112,7 +113,7 @@ changed:
             $AnimatedSprite.animation = "up"
             $AnimatedSprite.flip_v = velocity.y > 0
 
-    func _on_Player_body_entered( body ):
+    func _on_player_body_entered(body):
```

Comment:
It's named with a capital "P" in the original project, so changing it will confuse people. The node is named "Player" so when you connect the signal and Godot automatically creates the function, it uses that name.

Reply:
Sorry, missed the capitalization change, only saw the deleted spaces